arch/arm/boot/dts/overlays/README

```diff
@@ -273,8 +273,21 @@ Params: <None>
 
 Name:   hifiberry-dacplus
 Info:   Configures the HifiBerry DAC+ audio card
-Load:   dtoverlay=hifiberry-dacplus
-Params: <None>
+Load:   dtoverlay=hifiberry-dacplus,<param>=<val>
+Params: 24db_digital_gain       Allow gain to be applied via the PCM512x codec
+                                Digital volume control. Enable with 
+                                "dtoverlay=hifiberry-dacplus,24db_digital_gain"
+                                (The default behaviour is that the Digital
+                                volume control is limited to a maximum of
+                                0db. ie. it can attenuate but not provide
+                                gain. For most users, this will be desired 
+                                as it will prevent clipping. By appending 
+                                the 24db_digital_gain parameter, the Digital
+                                volume control will allow up to 24db of
```

Comment:
You might want to write `dB` (capital "B"), as is [usually done](https://en.wikipedia.org/wiki/Decibel) for the unit decibel. This would apply to several places in your patch.